cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -65,270 +101,267 @@ spec:
-        #       operator: Exists
-        # (Optional) Rook Agent priority class name to set on the pod(s)
-        # - name: AGENT_PRIORITY_CLASS_NAME
-        #   value: "<PriorityClassName>"
-        # (Optional) Rook Agent NodeAffinity.
-        # - name: AGENT_NODE_AFFINITY
-        #   value: "role=storage-node; storage=rook,ceph"
-        # (Optional) Rook Agent mount security mode. Can by `Any` or `Restricted`.
-        # `Any` uses Ceph admin credentials by default/fallback.
-        # For using `Restricted` you must have a Ceph secret in each namespace storage should be consumed from and
-        # set `mountUser` to the Ceph user, `mountSecret` to the Kubernetes secret name.
-        # to the namespace in which the `mountSecret` Kubernetes secret namespace.
-        # - name: AGENT_MOUNT_SECURITY_MODE
-        #   value: "Any"
-        # Set the path where the Rook agent can find the flex volumes
-        # - name: FLEXVOLUME_DIR_PATH
-        #   value: "<PathToFlexVolumes>"
-        # Set the path where kernel modules can be found
-        # - name: LIB_MODULES_DIR_PATH
-        #   value: "<PathToLibModules>"
-        # Mount any extra directories into the agent container
-        # - name: AGENT_MOUNTS
-        #   value: "somemount=/host/path:/container/path,someothermount=/host/path2:/container/path2"
-        # Rook Discover toleration. Will tolerate all taints with all keys.
-        # Choose between NoSchedule, PreferNoSchedule and NoExecute:
-        # - name: DISCOVER_TOLERATION
-        #   value: "NoSchedule"
-        # (Optional) Rook Discover toleration key. Set this to the key of the taint you want to tolerate
-        # - name: DISCOVER_TOLERATION_KEY
-        #   value: "<KeyOfTheTaintToTolerate>"
-        # (Optional) Rook Discover tolerations list. Put here list of taints you want to tolerate in YAML format.
-        # - name: DISCOVER_TOLERATIONS
-        #   value: |
-        #     - effect: NoSchedule
-        #       key: node-role.kubernetes.io/controlplane
-        #       operator: Exists
-        #     - effect: NoExecute
-        #       key: node-role.kubernetes.io/etcd
-        #       operator: Exists
-        # (Optional) Rook Discover priority class name to set on the pod(s)
-        # - name: DISCOVER_PRIORITY_CLASS_NAME
-        #   value: "<PriorityClassName>"
-        # (Optional) Discover Agent NodeAffinity.
-        # - name: DISCOVER_AGENT_NODE_AFFINITY
-        #   value: "role=storage-node; storage=rook, ceph"
-        # Allow rook to create multiple file systems. Note: This is considered
-        # an experimental feature in Ceph as described at
-        # http://docs.ceph.com/docs/master/cephfs/experimental-features/#multiple-filesystems-within-a-ceph-cluster
-        # which might cause mons to crash as seen in https://github.com/rook/rook/issues/1027
-        - name: ROOK_ALLOW_MULTIPLE_FILESYSTEMS
-          value: "false"
+        - name: rook-ceph-operator
```

Comment:
Can you avoid the whitespace changes? This will reduce the scope of the change.

Reply:
Sure.